src/xhydro/PMP.py

```diff
@@ -0,0 +1,609 @@
+"""Module to compute Probable Maximum Precipitation (PMP)"""
+
+import sys
+from itertools import product
+
+import dask
+import numpy as np
+import pandas as pd
+import xarray as xr
+import xclim
+from lmoments3.distr import gev
```

Comment:
Ok, I have bad news here: If we place lmoments3 directly into `xhydro`, we have to change the licensing for the project. Is there any way that you can rewrite the function using this callable to allow a lmoments3 object to be passed to it by the user?

We needed to do the same thing for `xclim`,